<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_head">
     <property name="text">
      <string>Fusion reaction in coupled channel method(CCFULL ) </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_kin">
         <property name="title">
          <string>Kinematics</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Projectile</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEdit_ap">
            <property name="text">
             <string>16.0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Energy</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>mass(u)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>emin(MeV)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLabel" name="label_emax">
            <property name="text">
             <string>emax</string>
            </property>
           </widget>
          </item>
          <item row="3" column="6">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>dE</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>charge</string>
            </property>
           </widget>
          </item>
          <item row="3" column="7">
           <widget class="QLineEdit" name="lineEdit_dE">
            <property name="text">
             <string>1.0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLineEdit" name="lineEdit_emax">
            <property name="text">
             <string>72.0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lineEdit_at">
            <property name="text">
             <string>144.0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLineEdit" name="lineEdit_zp">
            <property name="text">
             <string>8.0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>mass(u)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>charge</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEdit_emin">
            <property name="text">
             <string>55.0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Target</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLineEdit" name="lineEdit_zt">
            <property name="text">
             <string>62.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_pot">
         <property name="title">
          <string>Central Nuclear Potential</string>
         </property>
         <layout class="QGridLayout" name="layout_pot">
          <item row="0" column="4">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>a0(fm)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEdit_R0">
            <property name="text">
             <string>1.1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLineEdit" name="lineEdit_a0">
            <property name="text">
             <string>0.75</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>depth(MeV)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>R0(fm)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_V0">
            <property name="text">
             <string>105.1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QPushButton" name="pushButton_AWpot">
            <property name="text">
             <string>A-W pot</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Rmatch</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_rmatch">
            <property name="text">
             <string>50.0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>step size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEdit_rstep">
            <property name="text">
             <string>0.05</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_coup">
       <property name="title">
        <string>Coupling info</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_coup">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_proj_ex">
          <property name="title">
           <string>Proj. Excit.</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>RP</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_Rp">
             <property name="text">
              <string>1.2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Excit. type</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QComboBox" name="comboBox_proj_ex">
             <item>
              <property name="text">
               <string>inert</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>vibrational</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rotational</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_targ_ex">
          <property name="title">
           <string>Targ. Excit.</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="2">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Excit. type</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_Rt">
             <property name="text">
              <string>1.06</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>RT</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QComboBox" name="comboBox_targ_ex">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <item>
              <property name="text">
               <string>inert</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>vibrational</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rotational</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_transf">
       <property name="title">
        <string>Transfer Coupling between ground states(not available yet)</string>
       </property>
       <layout class="QHBoxLayout" name="layout_transf">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>NTRANS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_ntrans">
          <item>
           <property name="text">
            <string>no transfer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>transfer</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>QTrans</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_qtrans">
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>FTR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ftrans">
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
